examples/saul/main.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Example for demonstrating SAUL and the SAUL registry
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "shell.h"
+
+int main(void)
+{
+    puts("Welcome to RIOT!\n");
+    puts("Type `help` for help, type `saul` to see all SAUL devices\n");
+
+    char line_buf[SHELL_DEFAULT_BUFSIZE];
```

Comment:
hm, would make sense in general, but ALL example/test applications using the shell are doing it this way currently -> how about a separate PR that changes this for all effected apps?